pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+| `not`                       | Core          | 04                 | No      | No                                                                                           |
+| `oneOf`                     | Core          | 04                 | No      | No                                                                                           |
+| `optional`                  | Core          | 02                 | No      | Replaced by `required`                                                                       |
+| `pattern`                   | Core          | 01                 | No      | No                                                                                           |
+| `patternProperties`         | Core          | 03                 | No      | No                                                                                           |
+| `prefixItems`               | Core          | 2020-12            | No      | Replaced `items`                                                                             |
+| `propertyNames`             | Core          | 06                 | No      | No                                                                                           |
+| `readOnly`                  | Validation    | 01                 | No      | No                                                                                           |
+| `required`                  | Validation    | 03                 | No      | No                                                                                           |
+| `requires`                  | Core          | 01                 | 03      | Replaced by `dependencies`                                                                   |
+| `title`                     | Validation    | 01                 | No      | No                                                                                           |
+| `then`                      | Core          | 07                 | No      | No                                                                                           |
+| `type` (legacy)             | Core          | 01                 | 04      | Replaced by `type`                                                                           |
+| `type`                      | Validation    | 04                 | No      | Replaced `type` (legacy)                                                                     |
+| `unevaluatedItems`          | Core          | 2019-09            | No      | No                                                                                           |
+| `unevaluatedProperties`     | Core          | 2019-09            | No      | No                                                                                           |
+| `uniqueItems`               | Validation    | 02                 | Yes     | No                                                                                           |
```

Comment:
This keyword was never removed, though it says "Yes" here

Reply:
Noted and applied.